spec/models/guide_spec.rb

```diff
@@ -11,11 +11,64 @@
 
   describe "validations" do
     it "doesn't allow slugs without /service-manual/ prefix" do
-      edition = Generators.valid_edition(title: "something", state: "published")
+      edition = Generators.valid_published_edition(title: "something", state: "published")
       edition = Guide.new(slug: "/something", latest_edition: edition)
       edition.valid?
       expect(edition.errors.full_messages_for(:slug)).to eq ["Slug must be be prefixed with /service-manual/"]
     end
   end
 
+  context "review and approval" do
+    let :edition do
+      edition = Generators.valid_edition
+      allow(edition).to receive(:persisted?) { true }
```

Comment:
you seem to be overriding this line in a few tests, perhaps you can move this stub to a before block in contexts where it's needed only